src/components/calcite-tab-title/calcite-tab-title.tsx

```diff
@@ -78,7 +78,7 @@ export class CalciteTabTitle {
 
   disconnectedCallback(): void {
     this.observer.disconnect();
-    this.calciteTabTitleUnregister.emit();
+    document.body.dispatchEvent(new CustomEvent("calciteTabTitleUnregister"));
```

Comment:
Maybe a comment on why this is different from other event emitting would be useful.